File: src/main/java/vocadb/notification/reader/client/query/TopSongsVocalist.java
package vocadb.notification.reader.client.query;

import com.fasterxml.jackson.annotation.JsonProperty;

public enum TopSongsVocalist {
    @JsonProperty("Nothing")
    NOTHING,
    @JsonProperty("Vocaloid")
    VOCALOID,
    @JsonProperty("UTAU")
    UTAU,
    @JsonProperty("CeVIO")
    CE_VIO;
}

File: src/main/java/vocadb/notification/reader/client/api/enums/TopSongsVocalist.kt
package vocadb.notification.reader.client.api.enums

import com.fasterxml.jackson.annotation.JsonValue

enum class TopSongsVocalist(@JsonValue val value: String) {
    NOTHING("Nothing"),
    VOCALOID("Vocaloid"),
    UTAU("UTAU"),
    CE_VIO("CeVIO")
}
